blocks/graph.py

```diff
@@ -354,3 +414,104 @@ def apply_noise(computation_graph, variables, level, seed=None):
         replace[variable] = (variable +
                              rng.normal(variable.shape, std=level))
     return computation_graph.replace(replace)
+
+
+def apply_dropout(computation_graph, variables, drop_prob=0.5, rng=None,
```

Comment:
I swear I saw a lot of different numbers in the literature. Similarly to the precedent with the transition of `SimpleRecurrent` I would not provide a default value here.